### DISCUSSION TITLE
Coded Arms - Boss cutscenes break mouse hooking

### DISCUSSION THREAD

**Discussion by Bloodino** [garungorp/MouseInjectorDolphinDuck#18] [General]
My first time posting anything in Github, so I hope someone will eventually read this:

Been able to play Coded arms fine at 30fps (I tested the 60 fps cheat, but the game was a tad too fast and wonky with that) using PPSSPP and the mouse injector (with DirectX11), however upon reaching any of the boss battles and the cutscene for them triggers, the mouse stops responding completely.

Moving and playing with a controller does seem to work perfectly fine, even the keybinds of the mouse and keyboard do respond, but no mouse movement whatsoever.
I tried hooking and unhooking and restarting the emulation to no avail
I've made save states to confirm that, reaching the level of the boss isn't the cause but rather it seems to be related to the small intro cutscene to the boss? (I'm unsure, only guessing here).

The serial ID of the rom corresponds to the one listed here. My PPSSPP version is: 1.17.1. Using the latest version of the mouse injector aswell.

I hope these infos can help. If there's anything else's for me to provide within' my ability to do so, please let me know.

**Comment by MiPHISTo** [comment]
I can confirm that this is an issue as well. Tried various troubleshooting steps such as beating the entire Ruins section again and as soon as the boss spawned the injection broke immediately. Created Save States and tried to reload them to no avail. Tried to die to the boss and see if restarting after doing so but nothing seemed to work. 

Likely as suggested it's the mini cutscene before each boss.
